examples/gno.land/r/gnoland/boards2/v1/README.md

```diff
@@ -0,0 +1,526 @@
+# Boards: Decentralized Discussion Forums on Gno.land
+
+Boards is a forum realm for Gno.land with robust role-based access control, moderation tools, and a clean public API. It serves two roles: a proof-of-concept that demonstrates building real dApps on Gno.land, and a practical tool for decentralized social coordination on the internet. It features pluggable permission systems, on-chain governance with irreversible commitments, moderation that balances openness with safety, and an event-driven architecture that enables integrations and observability. For communities, it provides transparent governance structures, on-chain decision-making, configurable moderation, permanent auditable records, and flexible role management that bridges on-chain governance with human dynamics.
+
+Access (v1): Boards operates invite-only access during beta. Creating boards and other privileged actions require realm membership. How to request/receive an invite: [TO ADD].
```

Comment:
Right now Boards2 exposes a [RequestInvite()](https://gno.land/r/gnoland/boards2/v1:admin-users$help#func-RequestInvite) function and a view that display invites requests that need approval thought the "Manage Board -> List Invite Requests" menu in each individual board.

I'm going add a top "Request Invite" link to each board.